src/components/Garden.js

```diff
@@ -1,136 +1,162 @@
-    .attr('x', 0)
-    .attr('y', 0)
-    .attr('transform', (d,i) => `translate(${100 + i * 200}, ${yStemScale(d.lifespan)})`)
-  petalGroup.append('path')
-    .attr('d', petalPathThree)
-    .attr('fill', d => colorsByLanguage[d.languages[2]] || 'none')
-    .attr('transform', `translate(-50, -180)`)
-  petalGroup.append('path')
-    .attr('d', petalPathThree)
-    .attr('fill', d => colorsByLanguage[d.languages[2]] || 'none')
-    .attr('transform', `translate(-50, -180)rotate(120, 50, 160)`)
-  petalGroup.append('path')
-    .attr('d', petalPathThree)
-    .attr('fill', d => colorsByLanguage[d.languages[2]] || 'none')
-    .attr('transform', `translate(-50, -180)rotate(240 50 160)`)
-  petalGroup.append('path')
-    .attr('d', petalPathTwo)
-    .attr('fill', d => colorsByLanguage[d.languages[1]] || 'none')
-    .attr('transform', `translate(-50, -150)rotate(60 50 130)`)
-  petalGroup.append('path')
-    .attr('d', petalPathTwo)
-    .attr('fill', d => colorsByLanguage[d.languages[1]] || 'none')
-    .attr('transform', `translate(-50, -150)rotate(180 50 130)`)
-  petalGroup.append('path')
-    .attr('d', petalPathTwo)
-    .attr('fill', d => colorsByLanguage[d.languages[1]] || 'none')
-    .attr('transform', `translate(-50, -150)rotate(300 50 130)`)
-  petalGroup.append('path')
-    .attr('d', petalPathOne)
-    .attr('fill', d => colorsByLanguage[d.languages[0]] || 'none')
-    .attr('transform', `translate(-50, -130)`)
-  petalGroup.append('path')
-    .attr('d', petalPathOne)
-    .attr('fill', d => colorsByLanguage[d.languages[0]] || 'none')
-    .attr('transform', `translate(-50, -130)rotate(120, 50, 110)`)
-  petalGroup.append('path')
-    .attr('d', petalPathOne)
-    .attr('fill', d => colorsByLanguage[d.languages[0]] || 'none')
-    .attr('transform', `translate(-50, -130)rotate(240,  50, 110)`)
-
-  //   DRAW STEMS
-  //   height scaled to activeLife
-    const stem = flowerBed.selectAll('.stem')
-      .data(repositories).enter()
-      .append('path')
-      .attr('class','stem')
-      .attr('id', (d, i) => `myStem${i}`)
-      .attr('d', d => `M0,600 C 80 ${yStemScale(d.lifespan)}, -20 ${yStemScale(d.lifespan)}, 0 ${yStemScale(d.lifespan)}`)
-      .attr('transform', (d, i) => `translate(${100 + i * 200},0)`)
-      .attr('stroke-width', 5)
-      .attr('stroke', 'green')
-      .attr('fill', 'none')
-
-      const animate = () => {
-        flowerBed.selectAll('.stem')
-          .transition().duration(2000)
-          .attr('d', d => `M0,600 C -20 ${yStemScale(d.lifespan)}, 20 ${yStemScale(d.lifespan)}, 0 ${yStemScale(d.lifespan)}`)
-            .attr('stroke', 'green')
-          .transition().duration(2000)
-          .attr('d', d => `M0,600 C 80 ${yStemScale(d.lifespan)}, -20 ${yStemScale(d.lifespan)}, 0 ${yStemScale(d.lifespan)}`)
-          .on("end", animate)
+      .range([gardenHeight - 300, gardenHeight - 350, gardenHeight - 400, gardenHeight - 450, gardenHeight/2, gardenHeight/3, gardenHeight/4])
```

Comment:
I think some of these are the same. GardenHeight - 400 is the same as gardenHeight/2 when it's set to be 800. This won't be the case if we change the height but right now I wonder what that's doing to our stems.

Reply:
oh yes, haha, you're right. However we do it, I just think we need to add more stem heights to the scale

Reply:
Latest commit fixes this